docker-compose.yml

```diff
@@ -274,6 +274,31 @@ services:
         - /var/run/docker.sock:/var/run/docker.sock:ro
         - /lib/modules:/lib/modules:ro
 
+    acme-mailcow:
+      depends_on:
+        - nginx-mailcow
+      image: mailcow/acme
+      build: ./data/Dockerfiles/acme
+      # All domains to be included in the certificate
+      command: ${MAILCOW_HOSTNAME}
+      #autodiscover.${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME}
+      environment:
+        - CONTAINERS_RESTART=mailcowdockerized_postfix-mailcow_1 mailcowdockerized_dovecot-mailcow_1 mailcowdockerized_nginx-mailcow_1
+      volumes:
+        - ./data/web/.well-known/acme-challenge:/var/www/acme:rw
+        - ./data/assets/ssl:/var/lib/acme/:rw
+        - /var/run/docker.sock:/var/run/docker.sock:ro
+      dns:
+        - 172.22.1.254
+      dns_search: mailcow-network
+      # TODO: find some way to use always
+      # do not restart the container after exit. Things get worse when we hit let's encrypt's ratelimit.
+      restart: "no"
```

Comment:
Is there any better restart-policy than `"no"`? Using `always` will result in hitting the ratelimit. This is much worse than a stopped container. If hit, the ratelimit cannot get reset manually and requires hours of time to wait. A stopped container can get started and the problem is fixed in seconds.